definitions/pipelines/immuno.cwl

```diff
@@ -50,6 +50,10 @@ inputs:
         type: File
     ribosomal_intervals:
         type: File
+    kallisto_kmer_size:
+        type: int?
```

Comment:
Now that this is determined inside `pipelines/rnaseq.cwl`, I believe this input can be removed here entirely.